### PR TITLE
Fix multi-word search

### PR DESCRIPTION
**Pull request by nsachin08**
Fix MULTI-KEYWORD search in AUTHORS PAGE.

### Testing

Seacrch for a book using more than 1 word in authors page.
### Screenshot

before1

after1

@mekarpeles  @tabshaikh

### CODE REVIEW

**Comment by cclauss** [comment]
https://travis-ci.com/github/internetarchive/openlibrary/jobs/454470742#L443

**Comment by cclauss** [comment]
When you type `make lint-diff` on your command line do you get output like
https://travis-ci.com/github/internetarchive/openlibrary/jobs/454576741#L443

**Comment by nsachin08** [comment]
> When you type `make lint-diff` on your command line do you get output like
> https://travis-ci.com/github/internetarchive/openlibrary/jobs/454576741#L443

its making some kind error
Screenshot from 2020-12-04 21-40-46

**Review by mekarpeles — APPROVED** [review]
lgtm

File: openlibrary/plugins/worksearch/code.py [hunk 1]
```diff
@@ -642,12 +642,20 @@ def works_by_author(akey, sort='editions', page=1, rows=100, has_fulltext=False,
...
-    solr_select = solr_select_url + "?fq=%s&q.op=AND&q=%s&start=%d&rows=%d&fl=%s&wt=json" % ('&fq='.join(fq), q, offset, rows, fl)
+    params = {
+        'fq': fq,
+        'q': q,
+        'start': offset,
+        'rows': rows,
+        'fl': fl,
+    }
+    solr_select = solr_select_url + "?fq=type:work&wt=json&q.op=AND&"
+    solr_select = solr_select_url + urllib.parse.urlencode(params, 'utf-8')
```
Comment (mekarpeles) [inline openlibrary/plugins/worksearch/code.py:656]: I think this is actually wrong -- we're clobbering `solr_select` and shouldn't be. We should be saying 
`solr_select += urllib.parse.urlencode(params, 'utf-8')`

Comment (cclauss) [reply openlibrary/plugins/worksearch/code.py:656]: `solr_select` and `solr_select_url` are two separate things so the += would not be right however I do believe that line 655 could safely be deleted.

**Pull request merged**